components/external-link/style.scss

```diff
@@ -0,0 +1,7 @@
+.new-window-icon {
+	.dashicon {
```

Comment:
A bit of horizontal margin might be nice, but is minor and can be addressed in a styling pass if necessary.